.github/PULL_REQUEST_TEMPLATE/v5.md

```diff
@@ -5,7 +5,7 @@ Please convert it to a draft if any of the following conditions are not met. Rev
 
 - [ ] Added tests for the changes
 - [ ] Changed lines covered in coverage report
-- [ ] Change log has been added to `changes/` dir
+- [ ] Change log has been added to `changes/<version>/(feat|fix)-<PR-id>.en.md` and `.zh.md` files
```

Comment:
I think we don't need to differentiate the change type in the filename, as the commits already enforce that, and the descriptions hopefully will do as well.

Reply:
Nevermind, now I see why we're differentiating them here.